test-cases/upgrades/generic-rolling-upgrade.yaml

```diff
@@ -0,0 +1,35 @@
+# Workloads
+# stress_before_upgrade:           # To be configured in additional yaml
+# stress_during_entire_upgrade:       # To be configured in additional yaml
+# stress_after_cluster_upgrade:    # To be configured in additional yaml
+
+n_db_nodes: 3
+n_loaders: 1
+n_monitor_nodes: 1
+
+num_nodes_to_rollback: 2   # max is n_db_nodes -1, after upgrading 4 nodes last node can't be rollback
+
+linux_distro: 'centos'
```

Comment:
```suggestion
scylla_linux_distro: 'centos'
```

Reply:
Didn't even know we have this.
What's the difference?
The original rolling-upgrade use linux_distro.

Reply:
I always was called `scylla_linux_disto` but in the pipeline it was called linux_distro for some reason... I guess it was me being lazy...